graylog2-web-interface/src/routing/Routes.ts

```diff
@@ -265,7 +265,7 @@ const Routes = {
   edit_extractor: (nodeId: string, inputId: string, extractorId: string) => `/system/inputs/${nodeId}/${inputId}/extractors/${extractorId}/edit`,
 
   edit_input_extractor: (nodeId: string, inputId: string, extractorId: string) => `/system/inputs/${nodeId}/${inputId}/extractors/${extractorId}/edit`,
-  getting_started: (fromMenu) => `${Routes.GETTING_STARTED}?menu=${fromMenu}`,
+  welcome: () => Routes.WELCOME,
```

Comment:
We can get rid of this function an just use `Routes.WELCOME` everywhere instead.